### ISSUE TITLE
TypeError while running basic winapps.list_installed

### ISSUE THREAD

**Issue by vibbs** [rominf/winapps#1]
Was trying the basic documentation and when trying to print apps in iterative way for the below code snippet getting the following error:

```
import winapps

for app in winapps.list_installed():
    print(app)
```

```
Traceback (most recent call last):
  File "C:/<path>/temp.py", line 20, in <module>
    for app in winapps.list_installed():
  File "C:\<path>\venv36\lib\site-packages\winapps\__init__.py", line 42, in <genexpr>
    return (application for application in application_or_none_generator if application is not None)
  File "C:\<path>\venv36\lib\site-packages\winapps\__init__.py", line 41, in <genexpr>
    for application_key in _installed_application_keys())
  File "C:\<path>\venv36\lib\site-packages\winapps\__init__.py", line 186, in _installed_application
    setattr(result, *f(value))
  File "C:\<path>\venv36\lib\site-packages\winapps\__init__.py", line 106, in <lambda>
    'install_date', _none_on_value_not_set(value) and datetime.strptime(value, '%Y%m%d').date()),
TypeError: strptime() argument 1 must be str, not int
```

Thanks in advance

**Comment by thieflord06** [comment]
I've had the same error...trying to work on a solution.

**Comment by rominf** [comment]
I'm sorry, I don't have a Windows machine for development right now.
PRs are welcome!

**Comment by thieflord06** [comment]
I was able to circumvent the date format error by getting rid of the check of the date "and datetime.strptime(value, '%Y%m%d').date())" from line 106 in the init.py. This is a dirty fix because I'm not sure what else is using this check.

**Comment by QuentiumYT** [comment]
This issue has been fixed on the latest pull request I've done #5, it now supports the 3 different type of dates you can retrieve.

**Comment by rominf** [comment]
Fixed in the latest commit. Please install it from GitHub, as I don't have access to the Windows machine.
